Vagrantfile

```diff
@@ -20,14 +21,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
   end
 
+  config.ssh.insert_key = false # for vagrant 1.8.5 see https://github.com/mitchellh/vagrant/issues/7610#issuecomment-234609660

```

Comment:
@smartbit 
the problem is **/home/vagrant/.ssh/authorized_keys** permission within the guest.

**Solution 1:**
manually insert the following line to ( /opt/vagrant/embedded/gems/gems/vagrant-1.8.5/plugins/guests/linux/cap/public_key.rb) after line 56 (mv ~/.ssh/authorized_keys.tmp ~/.ssh/authorized_keys):

chmod 0600 ~/.ssh/authorized_keys

**Solution 2:**
create patch file with the following contents
```shell
56a57
>               chmod 0600 ~/.ssh/authorized_keys
```
and apply the patch
```shell
$ sudo patch /opt/vagrant/embedded/gems/gems/vagrant-1.8.5/plugins/guests/linux/cap/public_key.rb < patch_file
```